compiler/rustc_const_eval/src/transform/check_consts/check.rs

```diff
@@ -896,6 +896,20 @@ impl Visitor<'tcx> for Checker<'mir, 'tcx> {
                         }
                     }
 
+                    // const-eval of the `panic_display` fn assumes the argument is `&&str`
```

Comment:
I know this is pre-existing, but it seems odd to specially check 2 out of the 4 panic functions here.

Reply:
Added a comment above